src/main/perl/MD.pm

```diff
@@ -55,10 +56,11 @@ sub _initialize
 
     my $st = $config->getElement($path)->getTree;
     $path=~m!/([^/]+)$!;
-    $self->{devname} = $1;
+    $self->{devname} = unescape($1);
     $st->{raid_level} =~ m!(\d)$!;
     $self->{raid_level} = $1;
     $self->{stripe_size} = $st->{stripe_size};
+    $self->{metadata} = ($st->{metadata}) ? $st->{metadata} : "0.90";
```

Comment:
This is to be backwards compatible. Right now metadata is hardcoded set to 0.90 (which is still needed for boot partitions iirc) , see https://github.com/quattor/ncm-lib-blockdevices/pull/56/files#diff-aad1758a837e49b5d4fd688343af9b72L336
so if it is not set, I would use the same value as before.. but I will make it optional in the schema